client/signup/steps/domains/index.jsx

```diff
@@ -243,6 +244,16 @@ const DomainsStep = React.createClass( {
 			);
 		}
 
+		let fallbackHeaderText,
+			fallbackSubHeaderText;
+		if ( abtest( 'signupDomainsHeadline' ) === 'updated' ) {
+			fallbackHeaderText = this.translate( 'Let\'s find a cool name for your brand new site' ),
+			fallbackSubHeaderText = this.translate( 'Choose a compelling name, it will be the address of your new site!' );
+		} else {
```

Comment:
You can probably skip the else and just assign defaults above, and change on `A/B` test. Your call.